app/models/ability.rb

```diff
@@ -54,13 +54,15 @@ def initialize(user, token = nil)
     owned_orgs = Organization.with_role(:owner, user).pluck(:id)
     can :manage, Organization, id: owned_orgs
     can %i[crud profile info], [Stream, Upload], organization: { id: owned_orgs }
+    cannot %i[reanalyze], [Stream], organization: { id: owned_orgs }
     can :read, MarcRecord, upload: { organization: { id: owned_orgs } }
     can :manage, AllowlistedJwt, resource_type: 'Organization', resource_id: owned_orgs
     can :read, ActiveStorage::Attachment, { record: { organization: { id: owned_orgs } } }
 
     member_orgs = Organization.with_role(:member, user).pluck(:id)
-    can %i[invite read users organization_details provider_details], Organization, id: member_orgs
+    can %i[invite read users organization_details provider_details normalized_data], Organization, id: member_orgs
     can %i[read profile info], [Stream, Upload], organization: { id: member_orgs }
+    can %i[create], [Upload], organization: { id: member_orgs }
```

Comment:
Not 100% sure what behavior we want?

Reply:
@corylown my reading of the permissions doc and comments in tickets was that members can create uploads, but not create streams. Did you have a different understanding?

Reply:
@corylown Hm just looked and I think this may have been changed in the doc. I can remove upload function from members?

Reply:
@marlo-longley I guess leave what you have. I'm not really sure what we want to allow members to do.